scripts/config/constants.ts

```diff
@@ -65,6 +66,7 @@ export const mainPackage: Config = {
 		coupons: {
 			packageId: '0x6d14ca3049be747ec87166e6dce5d0d9a30f3b3c281c55d6e518958a236f8b97',
 		},
+		treasuryAddress: '0x638791b625c4482bc1b917847cdf8aa76fe226c0f3e0a9b1aa595625989e98a1',
```

Comment:
confirmed this is the correct address
`0x638791b625c4482bc1b917847cdf8aa76fe226c0f3e0a9b1aa595625989e98a1`